src/components/badge/badge.module.css

```diff
@@ -1,28 +1,29 @@
 .root {
-  align-items: center;
-  border-color: transparent;
-  border-radius: 4px;
-  border-style: solid;
-  border-width: 1px;
-  display: inline-flex;
-  font-size: var(--font-size);
-  font-weight: 500;
-  justify-content: center;
-  line-height: var(--line-height);
-  min-height: var(--min-height);
-  padding: var(--padding-top-bottom) var(--padding-left-right);
-  width: fit-content;
-
-  /* HDS Badge has 12px icons when size="small" */
-  & svg {
-    height: var(--icon-size);
-    line-height: var(--line-height);
-    width: var(--icon-size);
-
-    &:not(:only-child) {
-      margin-right: var(--icon-margin);
-    }
-  }
+	align-items: center;
+	border-color: transparent;
+	border-radius: 4px;
+	border-style: solid;
+	border-width: 1px;
+	display: inline-flex;
+	font-family: var(--token-typography-font-stack-text);
```

Comment:
We have a `--font-body` variable, which is set at the `<Dialog>` level and the `CoreDevDotLayout` level to help with this.

I'm not sure if it makes sense to have another layer of custom properties, but it is available and used by a few other components (`Button` and `ButtonLink` right now).

Reply:
I think we're fine to use `--token-typography-font-stack-text` since the `--font-body` variable being set by `Dialog` and `CoreDevDotLayout` is primarily useful for external components that use those variables, like the `<button>` elements that are in `CodeTabs`.